modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,51 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.10]]
+== Version 5.10
+
+===  Updated features
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+IS [NOT] :: <TYPE>
+----
+a|
+
+Extended xref:syntax/expressions.adoc#type-predicate-expressions[type predicate expressions].
+The newly supported types are:
```

Comment:
```suggestion
The newly supported expressions are:
```

Reply:
the type is the `STRING` part, the list below is not repeating the whole expression only the type part of the expression